src/lephare/process.py

```diff
@@ -219,3 +222,78 @@ def table_to_data(config, input, col_names=None, standard_names=False):
     flux[mask] = -99.0
     flux_err[mask] = -99.0
     return id, flux, flux_err, context, zspec, string_data
+
+
+def load_sed_list(path_to_list, object_type, absolute_paths=False):
+    """Take a sed list and move it with the sed files to the work directory.
+
+    We want to be able to load SED lists from other sources. These may use
+    absolute paths or relative to the list file. They must adopt the lepahre
+    standard which is relative to the sed/$TYPE/ directory
+
+    Parameters
+    ==========
+    path_to_list : str
+        The path to the SED list file.
+    object_type : str
+        The type of object. Must be one of "STAR", "GAL", or "QSO".
+    absolute_paths : bool
+        Set to True if list of absolute paths and not relative to list file
+
+    Returns
+    =======
+        new_list_file : str
+            The location of the new list file with respect to current
+            LEPHAREWORK directory. This is the value to set
+    """
+    # The type must by one of the three lephare standards
+    assert object_type in ["STAR", "GAL", "QSO"]
+    # The original file must exist
+    assert os.path.isfile(path_to_list)
+
+    # Set folder and file names.
+    name = path_to_list.split("/")[-1].split(".")[0]
+    old_dir = os.path.dirname(path_to_list)
+    new_dir = f"{lp.dm.lephare_dir}/sed/{object_type}/{name}"
```

Comment:
Might be better to use the `os.path.join` functionality here: 
```suggestion
    new_path = os.path.join(lp.dm.lephare_dir, "sed", f"{object_type}/{name}.list")
    new_dir = os.path.dirname(new_path)
    new_list_file = os.path.basename(new_path)
```